frontend/src/components/DocumentUnitList.vue

```diff
@@ -342,13 +327,28 @@ function onDelete() {
                   params: { documentNumber: listEntry.documentNumber },
                 }"
               >
-                <IconView />
+                <Button
+                  aria-label="Dokumentationseinheit ansehen"
+                  class="z-20"
+                  severity="secondary"
+                  size="small"
+                >
+                  <template #icon>
+                    <IconView />
+                  </template>
+                </Button>
               </router-link>
             </Tooltip>
-            <Tooltip v-if="listEntry.isDeletable" text="Löschen">
-              <button
+            <Tooltip text="Löschen">
+              <Button
```

Comment:
Tooltip in letzter Reihe ist nicht sichtbar
<img width="185" alt="image" src="https://github.com/user-attachments/assets/68a22bb0-6ac1-4dc2-b6fe-c4f7f0108b35" />

Reply:
ist hier angepasst: https://github.com/digitalservicebund/ris-backend-service/commit/88e8eb224c6aa7a907f466c268b214af5b3cd795